daft/series.py

```diff
@@ -458,3 +462,10 @@ def day_of_week(self) -> Series:
 class SeriesArrayNamespace(SeriesNamespace):
     def lengths(self) -> Series:
         return Series._from_pyseries(self._series.arr_lengths())
+
+
+class SeriesImageNamespace(SeriesNamespace):
+    def decode(self, image_format: str | ImageFormat | None = None) -> Series:
```

Comment:
We should rely on the bytes to decode the image.

Reply:
I added this as an optional thing in case the user doesn't want to rely on our/`image`'s content-based format deduction, but I just checked and it looks like content-based format detection is reliable for the formats that I'd imagine we'd start with (PNG, JPEG, TIFF, BMP, GIF).

I'll remove this as `image_format` option for now, and if content-based format deduction doesn't work for a particular format, we can add it back.